python/lwt_interface/tskit_lwt_interface.h

```diff
@@ -1339,6 +1402,21 @@ parse_table_collection_dict(tsk_table_collection_t *tables, PyObject *tables_dic
         goto out;
     }
 
+    /* index */
+    value = get_table_dict_value(tables_dict, "index", false);
```

Comment:
Probably best to call this "indexes" just for consistency with the C struct?

Reply:
For consistency we should also change the python property on `TableCollection` to `indexes` but then that is inconsistent with `build_index` and `has_index` which would be breaking changes to move to `indexes`... we may have to live with the python API being `index` and the c struct being `indexes`.